usersession/agent/rest_api.go

```diff
@@ -324,8 +324,8 @@ func postRefreshFinishedNotification(c *Command, r *http.Request) Response {
 		})
 	}
 
-	summary := fmt.Sprintf(i18n.G("%q snap has been refreshed"), finishRefresh.InstanceName)
-	body := i18n.G("Now available to launch")
+	summary := fmt.Sprintf(i18n.G("“%q” snap refreshed"), finishRefresh.InstanceName)
+	body := i18n.G("Ready to launch")
```

Comment:
I think this was just changed because there was consensus on this message https://github.com/snapcore/snapd/pull/12715/files#r1166975484

Reply:
Ok...